### DISCUSSION TITLE
Adding publish image support for images uploaded in Excel sheets

### DISCUSSION THREAD

**Discussion by absarasw** [adobe/helix-home#353] [Ideas]
Currently images upload feature has only been supported through docs, franklin-bot and sharepoint/gdrive upload. But since excels are also a medium of authoring in Franklin, we should support uploading images through excels as well. 

**Need**
We are working on a project where we would like use excel sheets to accumulate informations from various stakeholders which include uploading images. So, we want the franklin pipeline to publish the images uploaded in the excel sheet and provide the URL in the json. 

**Workaround**
Right now we have to either upload the image in a doc or through franklin-bot to get the image URL but this seems to be an extra step which we would like to avoid.

**Comment by tripodsan** [comment]
see https://github.com/adobe/helix-admin/issues/354

**Reply by absarasw** [reply]
thanks @tripodsan. I didn't see any conclusion over there. Did we drop the idea or is it still in pipeline?
